packages/run-protocol/test/vaultFactory/test-vaultFactory.js

```diff
@@ -544,27 +544,62 @@ test('price drop', async t => {
   trace('price changed to liquidate', notification.value.vaultState);
   t.is(notification.value.vaultState, Phase.LIQUIDATING);
 
+  t.deepEqual(
+    await E(vault).getCollateralAmount(),
+    AmountMath.makeEmpty(aethBrand),
+    'Collateral consumed while liquidating',
+  );
+  t.deepEqual(
+    await E(vault).getCurrentDebt(),
+    AmountMath.make(runBrand, 284n),
+    'Debt remains while liquidating',
+  );
+
   await manualTimer.tick();
   notification = await E(vaultNotifier).getUpdateSince(
     notification.updateCount,
   );
   t.falsy(notification.updateCount);
   t.is(notification.value.vaultState, Phase.LIQUIDATED);
 
+  t.truthy(await E(vaultSeat).hasExited());
+
   const debtAmountAfter = await E(vault).getCurrentDebt();
   const finalNotification = await E(vaultNotifier).getUpdateSince();
   t.is(finalNotification.value.vaultState, Phase.LIQUIDATED);
+  t.deepEqual(finalNotification.value.locked, AmountMath.make(aethBrand, 1n));
   t.truthy(AmountMath.isEmpty(debtAmountAfter));
 
   t.deepEqual(await E(vaultFactory).getRewardAllocation(), {
     RUN: AmountMath.make(runBrand, 14n),
   });
 
-  const liquidations = await E(
-    E(vault).getLiquidationSeat(),
-  ).getCurrentAllocation();
-  t.deepEqual(liquidations.Collateral, AmountMath.make(aethBrand, 1n));
-  t.deepEqual(liquidations.RUN, AmountMath.makeEmpty(runBrand));
+  const emptyPurse = await E(services.runKit.issuer).makeEmptyPurse();
+  /** @type {UserSeat<string>} */
+  const closeSeat = await E(zoe).offer(
+    E(vault).makeCloseInvitation(),
+    harden({
+      give: { RUN: AmountMath.makeEmpty(runBrand) },
+      want: { Collateral: AmountMath.makeEmpty(aethBrand) },
+    }),
+    harden({
+      RUN: await E(emptyPurse).withdraw(AmountMath.makeEmpty(runBrand)),
```

Comment:
You shouldn't have to include an empty payment here.  I thought there would be a way to tell Zoe, but the contract is insisting that the payment has to be present. I'd consider that a bug in the contract. Vault's `closeHook` should allow you to close the vault without sending an empty payment.

Reply:
Great. That's the "better way" I was looking for. I've simplifed this to `const closeSeat = await E(zoe).offer(E(vault).makeCloseInvitation())` by only expecting a give/want proposal for when the vault is ACTIVE (ignoring the proposal shape when LIQUIDATED).